frontend/amundsen_application/static/js/ducks/tableMetadata/api/helpers.ts

```diff
@@ -3,13 +3,15 @@ import * as qs from 'simple-query-string';
 import { filterFromObj } from 'ducks/utilMethods';
 
 import {
+  NestedTableColumn,
   NotificationType,
-  PeopleUser,
+  PeopleUser, TableColumn, TableColumnType,
```

Comment:
The commit hook didn't work for you?
You might need to nuke node_modules and install again